sql/upgrade/sl3.0.sql

```diff
@@ -896,19 +920,24 @@ UPDATE business_unit_class
  WHERE id = 2
    AND EXISTS (select 1 from :slschema.project);
 
+ALTER TABLE gl DISABLE TRIGGER gl_audit_trail;
+ALTER TABLE gl DISABLE TRIGGER gl_prevent_closed;
+
 INSERT INTO gl(id, reference, description, transdate, person_id, notes)
     SELECT gl.id, reference, description, transdate, p.id, gl.notes
       FROM :slschema.gl
  LEFT JOIN :slschema.employee em ON gl.employee_id = em.id
  LEFT JOIN person p ON em.entity_id = p.id;
 
+ALTER TABLE gl ENABLE TRIGGER gl_prevent_closed;
```

Comment:
That way, you can't forget to re-enable it :smile:

Reply:
Removed